### PR TITLE
copied assignment to new repo and modified as per previous feed back …

### PR DESCRIPTION
**Pull request by M-Madhavreddy**
…along with exception handling methods

### CODE REVIEW

File: src/test/scala/m_madhav_reddy/BasicCalculatorTesting.scala [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    assert(basicCalculator.divisionWithOption(5, 5).isDefined)
+
+    //TestCase for Exception Handling using Either
+    assert(basicCalculator.divisionWithEither(5, 0).isLeft)
+    assert(basicCalculator.divisionWithEither(5, 5).isRight)
+
+  }
+
+
+
```
Comment (tg-trainer) [inline src/test/scala/m_madhav_reddy/BasicCalculatorTesting.scala:58]: remove empty lines

**Review by tg-trainer — CHANGES_REQUESTED** [review]
Please check your spacing

**Comment by M-Madhavreddy** [comment]
changed as per request

**Review by M-Madhavreddy — COMMENTED** [review]
Removed extra space

**Pull request merged**